src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -1,59 +1,101 @@
 package org.roy.loadx.transaction;
 
 import java.util.ArrayList;
-import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
 
 public class TransactionPrinter {
-	private final TransactionAggregator transactionAggregator;
+	static private class TransactionDataSummaryEntry {
+		private String transactionName;
+		private String average;
+		private String count;
+		
+		public TransactionDataSummaryEntry(String transactionName, String average, String count) {
+			this.transactionName = transactionName;
+			this.average = average;
+			this.count = count;
+		}
 
-	public TransactionPrinter(TransactionAggregator transactionAggregator) {
-		this.transactionAggregator = transactionAggregator;
-	}
+		public String getTransactionName() {
+			return transactionName;
+		}
 
-	private static void println() {
-		System.out.println();
-		System.out.flush();
+		public String getAverage() {
+			return average;
+		}
+
+		public String getCount() {
+			return count;
+		}
 	}
+	
+	static private class TransactionDataSummary {
+		static private final String TRANSACTION_HEADER = "Transaction";
+		static private final String AVERAGE_HEADER = "Average";
+		static private final String COUNT_HEADER = "Count";
+		static private final int MAX_ROW_WIDTH = 78;
```

Comment:
I stand corrected, even on Windows you only need to account one char for the line feed (wrote a groovy test to run on Windows since my dev box is Linux and didn't want to configure a full env for this).  This code uses `System.out` under the hood.

```
(1..120).each {
	print it % 10 ? 'x' : '0'
}
println()
print 'New Row'
```

![width](https://cloud.githubusercontent.com/assets/351737/14804975/e13fca3c-0b35-11e6-8d1f-930a828a80ae.PNG)

120 + newline = wrap
119 + newline = no wrap, even on Windows

Updated ROW_WIDTH to 119.